Detectors/FIT/FV0/base/include/FV0Base/Geometry.h

```diff
@@ -120,6 +124,14 @@ class Geometry
   /// \return True if cellId belongs to ring 5.
   bool isRing5(UInt_t cellId);
 
+  static constexpr o2::detectors::DetID::ID getDetID() { return o2::detectors::DetID::FV0; }
+  TGeoPNEntry* getPNEntry(int index) const
+  {
+    /// Get a pointer to the TGeoPNEntry of a chip identified by 'index'
+    /// Returns NULL in case of invalid index, missing TGeoManager or invalid symbolic name
+    return o2::base::GeometryManager::getPNEntry(getDetID(), index);
+  }
```

Comment:
Looks fine, but why do you need this aliases, which are anyway never used?

Reply:
May be in future FV0 will want to use it